### PR TITLE
set up a test suite for the backend. closes #52

### PR DESCRIPTION
**Pull request by aolundin**
Still have to create tests for deleting and getting single attributes. closes #52

### CODE REVIEW

**Comment by aolundin** [comment]
closes #44

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        "name": "test1",
+        "createdAt": "2015-06-14T13:08:12.348Z",
+        "updatedAt": "2015-06-14T13:08:12.348Z"
+    }];
+
+    var couchdb = nock(mockedUrl, {allowUnmocked: true})
+        .get('/attribute')
+        .reply(200, resultAllGet);
+
+    it('should reply with HTTP status code 200 and a correctly formatted JSON object when making a GET request to /api/attribute', function(done) {
```
Comment (Tobbe) [inline test/test.js:34]: Remove url from end of string

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        "_id": "557fd13a9a81250f00194d58"
+    };
+
+    var couchdb = nock(mockedUrl, {allowUnmocked: true})
+    .post('/attribute', {
+        name: 'test1'
+    })
+    .reply(200, resultPost);
+
+    it('should reply with HTTP status code 200 and a correctly formatted JSON object when making a POST request to /api/attribute', function(done) {
```
Comment (Tobbe) [inline test/test.js:66]: Remove url from end of string

**Comment by aolundin** [comment]
ill-named issue. only meant as a proof of concept. Stay tuned for more

**Pull request merged**